_health-care/_js/_form.js

```diff
@@ -1284,6 +1333,7 @@ var HealthApp = (function() {
     xmlFieldMap: xmlFieldMap,
     getFormRoot: getFormRoot,
     saveForm: saveForm,
+    validate: validate,
```

Comment:
That is the correct way.  We might want to add another layer in the future that designates the API is exposed for test like
```
{
      xmlFieldMap: xmlFieldMap,
      getFormRoot: getFormRoot,
      saveForm: saveForm,
      forTest: {
        validate: validate
     }
}
```